cypress/support/fragments/settings/circulation/fee-fine/overdueFinePolicies.js

```diff
@@ -0,0 +1,174 @@
+import { Heading, Link, Button, Section, PaneHeader, PaneContent, TextField, HTML, Select, KeyValue } from '../../../../../../interactors';
+import getRandomPostfix from '../../../../utils/stringTools';
+
+const mainTitle = 'Overdue fine policies';
+const createTitle = 'New overdue fine policy';
+const actionsButton = Button({ id: 'clickable-create-entry' });
+const cancelButton = Button({ id: 'footer-cancel-entity' });
+const saveButton = Button({ id: 'footer-save-entity' });
+const createFormSection = Section({ id: 'overdue-fine-policy-pane' });
+const createFormHeader = PaneHeader({ id: 'paneHeaderoverdue-fine-policy-pane' });
+const createFormContent = PaneContent({ id: 'overdue-fine-policy-pane-content' });
+const overdueFineInput = TextField({ name:'overdueFine.quantity' });
+const generalInformationButton = Button({ id: 'accordion-toggle-button-overdueGeneralSection' });
+const nameInput = TextField('Overdue fine policy name*');
+const generalInformationSection = Section({ id: 'overdueGeneralSection' });
+const nameRequiredAlert = 'Please fill this in to continue';
+const name = `Autotest_fine_policy_name_${getRandomPostfix()}`;
+const editFineSection = Section({ id: 'editFineSection' });
+const overdueFineNegativeAlert = 'The value must be greater than or equal to 0';
+const maximumOverDueFineAlert = 'Maximum overdue fine must be greater than or equal to Overdue fine';
+const intervalAlert = 'Please make a selection';
+const intervalSetAlert = 'Overdue fine must be greater than 0 if interval set';
+const alertOverdueAmount = 'Please enter an overdue fine amount or set maximum overdue fine to 0';
+const intervalSelect = Select({ name: 'overdueFine.intervalId' });
+const generalFeePolicySection = Section({ id: 'generalFeePolicy' });
+const feeViewSection = Section({ id: 'viewFineSection' });
+
+export default {
+  waitLoading() {
+    cy.expect([
+      Link(mainTitle).exists(),
+      Heading(mainTitle).exists(),
+    ]);
+  },
+
+  openCreatingForm() {
+    cy.do(actionsButton.click());
+  },
+
+  checkCreatingForm() {
+    cy.do(createFormSection.click());
+    cy.expect([
+      createFormSection.exists(),
+      createFormHeader.exists(),
+      createFormContent.exists(),
+      cancelButton.has({ disabled : false }),
+      saveButton.has({ disabled : true }),
+      createFormSection.find(Heading(createTitle)).exists(),
+      generalInformationButton.exists(),
+      nameInput.exists(),
+      generalInformationSection.find(HTML(nameRequiredAlert)).exists(),
+      cy.expect(overdueFineInput.has({ value : '0.00' }))
+    ]);
+  },
+
+  fillOverdueFine(value) {
+    cy.get('[name="overdueFine.quantity"]').invoke('val', '').clear()
```

Comment:
Fix linter issues

Reply:
if i fix it - it won't work. i'll leave a comment

Reply:
@kimkate303 Please add this `// eslint-disable-next-line cypress/no-force` at the top of the cy.get.
```suggestion
    // eslint-disable-next-line cypress/no-force
    cy.get('[name="overdueFine.quantity"]').invoke('val', '').clear()
```